cdc/puller/ddl_puller.go

```diff
@@ -597,10 +598,7 @@ func (h *ddlPullerImpl) Run(ctx context.Context) error {
 	ctx, cancel := context.WithCancel(ctx)
 	h.cancel = cancel
 
-	ctx = contextutil.PutTableInfoInCtx(ctx, -1, DDLPullerTableName)
-	ctx = contextutil.PutChangefeedIDInCtx(ctx, h.changefeedID)
```

Comment:
Does it cause empty changefeed id in grafana by removing this line? Could you remove `PutChangefeedIDInCtx` in `contextutil` package?

Reply:
no, it's passed explicitly by parameters.

Reply:
`PutChangefeedIDInCtx` cannot be removed at the moment, since it's used by other files.